src/sentry/static/sentry/app/components/pill.jsx

```diff
@@ -29,19 +29,80 @@ class Pill extends React.Component {
   };
 
   render() {
-    const {name, children, className, ...props} = this.props;
-    const [extraClass, renderedValue] = this.renderValue();
+    const {name, children, ...props} = this.props;
+    const [renderedValue] = this.renderValue();
```

Comment:
This is not what you want... when you destructure an array, the position is important.

The old code said this:

```javascript
const foo = this.renderValue(); // This is an array of [<extraClass>, <renderedValue>]
const extraClass = foo[0];
const renderedValue = foo[1];
```

new code:

```javascript
const foo = this.renderValue();
const renderedValue = foo[0];
```

Reply:
@billyvg Is the best course of action to just revert this back to what it was before?

Reply:
yeah

Reply:
I assume you removed it because eslint was yelling (since it's not being used) (but it should be used) (but also to have eslint ignore something, prepend with a `_`)